app/serializers/event_serializer.rb

```diff
@@ -5,7 +5,7 @@ class EventSerializer < ActiveModel::Serializer
 
   def date
     t = object.start_time
-    t.blank? ? '' : %{ #{l t, format: :short}#{t.formatted_offset(false)} }
+    t.blank? ? '' : %{ #{I18n.l t, format: :short}#{t.formatted_offset(false)} }
```

Comment:
I think you can remove t.formatted_offset(false) and add it to :short  by adding ":z" .I'm not sure, so please do verify they give the same output.